spec/acceptance/user_journey_spec.rb

```diff
@@ -34,7 +34,7 @@ def sign_in_with_mfa
   end
 
   def rotate_msa_encryption_certificate
-    click_link t('user_journey.encryption_certificate')
+    find('.govuk-table__row', text: 'IN USE', match: :first).click_link
```

Comment:
is it not going to click a signing cert?

Reply:
@jakubmiarka thanks for your help with this :) I have amended.